intranet/static/js/eighth/schedule.js

```diff
@@ -175,5 +176,18 @@ $(function() {
         if (activities !== "" && !confirm("Are you sure you want to add the following activities without a sponsor?\n" + activities)) {
             e.preventDefault();
         }
+        var stickyCancel = "";
+        $("tr.form-row.scheduled:not(.hidden)").each(function() {
```

Comment:
```suggestion
        $("tr.form-row.scheduled").each(function() {
```

This proved necessary during testing.